ludwig/encoders/h3_encoders.py

```diff
@@ -42,70 +48,49 @@ class H3Embed(H3Encoder):
+                   for `dense` representations and exactly `vocabulary_size`
                    for the `sparse` encoding, where `vocabulary_size` is
                    the number of different strings appearing in the training set
-                   in the column the feature is named after (plus 1 for `<UNK>`).
+                   in the column the feature is named after (plus 1 for
+                   `<UNK>`).
             :type embedding_size: Integer
-            :param embeddings_on_cpu: by default embedings matrices are stored
+            :param embeddings_on_cpu: by default embeddings matrices are stored
                    on GPU memory if a GPU is used, as it allows
                    for faster access, but in some cases the embedding matrix
                    may be really big and this parameter forces the placement
-                   of the embedding matrix in regular memroy and the CPU is used
+                   of the embedding matrix in regular memory and the CPU is used
                    to resolve them, slightly slowing down the process
                    as a result of data transfer between CPU and GPU memory.
             :param dropout: determines if there should be a dropout layer before
                    returning the encoder output.
             :type dropout: Boolean
-            :param initializer: the initializer to use. If `None`, the default
-                   initialized of each variable is used (`glorot_uniform`
-                   in most cases). Options are: `constant`, `identity`, `zeros`,
-                    `ones`, `orthogonal`, `normal`, `uniform`,
-                    `truncated_normal`, `variance_scaling`, `glorot_normal`,
-                    `glorot_uniform`, `xavier_normal`, `xavier_uniform`,
-                    `he_normal`, `he_uniform`, `lecun_normal`, `lecun_uniform`.
-                    Alternatively it is possible to specify a dictionary with
-                    a key `type` that identifies the type of initialzier and
-                    other keys for its parameters, e.g.
-                    `{type: normal, mean: 0, stddev: 0}`.
-                    To know the parameters of each initializer, please refer to
-                    TensorFlow's documentation.
-            :type initializer: str
-            :param regularize: if `True` the embedding wieghts are added to
-                   the set of weights that get reularized by a regularization
-                   loss (if the `regularization_lambda` in `training`
-                   is greater than 0).
-            :type regularize: Boolean
         """
         super().__init__()
         logger.debug(' {}'.format(self.name))
 
         self.embedding_size = embedding_size
         self.reduce_output = reduce_output
-        self.reduce_sequence = SequenceReducer(reduce_mode=reduce_output)
+        self.sum_sequence_reducer = SequenceReducer(reduce_mode=reduce_output)
```

Comment:
I prefer the old name, as which reduce operation to use is user specified, it could be something different than sum.

Reply:
Done.

Reply:
I kept the other sequence reducer in this file as sum_sequence_reducer since that one seems to be configured strictly with reduce_mode=sum.